tools/data_integrity/fix_candidate_age.php

```diff
@@ -17,10 +17,7 @@
  * @license  Loris license
  * @link     https://www.github.com/aces/Loris-Trunk/
  */
-require_once 'generic_includes.php';
-
-$config   = NDB_Config::singleton();
-$database = $config->getSetting('database');
+require_once __DIR__ . '/../generic_includes.php';
```

Comment:
@ridz1208 Sorry, one last comment. It might be good to explain clearly what the confirm option does. I guessed it but feels it could be better documented.